<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="c0016390" failures="38" tests="70" name="Rebill Test" time="5753.449" errors="0" timestamp="2021-02-09T23:01:07 UTC">
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod2" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod2() returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod2() returned a null value
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod3" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod3() returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod3() returned a null value
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testMethod3 -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod4" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod4() returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod4() returned a null value
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testMethod4 -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="47.713">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.019">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="122.342"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.296">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="47.476">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.135">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="121.567"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="124.275"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.724">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="123.882"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="126.968"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="47.597">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.496">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="121.446"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.374">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.519">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.282">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="121.499"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.537">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.434">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="123.349"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.911">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="47.06">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.361">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.157">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.516">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="122.639"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.134">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.446">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="122.268"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.278">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="123.872"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.758">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="122.237"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="124.39"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="122.839"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.487">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="128.158"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="136.499"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="47.466">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="49.822">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.99">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="124.043"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="125.003"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="128.659"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="48.119">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="48.174">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="126.937"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="123.402"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.296">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.849">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.731">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.136">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="47.175">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="122.136"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.603">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="125.842"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="46.715">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:1003)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:809)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:643)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="123.56"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="125.771"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="123.988"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="122.967"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="138.22"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="159.269"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="134.389"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="139.036"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="136.094"/>
</testsuite> <!-- Rebill Test -->
